Update upgrading-kibana.asciidoc (#1997)
elastic/stack-docs: docs/en/install-upgrade/upgrading-kibana.asciidoc

. Start {kib}.

> IMPORTANT: {kib} has a new logging system in 8.0 and the {kibana-ref}/upgrade.html#logging-configuration-changes[log formats have changed]. 

Author: ```suggestion
IMPORTANT: {kib} has a new logging system in 8.0 and the {kibana-ref}/_upgrading_multiple_kibana_instances.html#logging-config-changes[log formats have changed]. 
```

> IMPORTANT: {kib} has a new logging system in 8.0 and the {kibana-ref}/_upgrading_multiple_kibana_instances.html#logging-config-changes[log formats have changed]. 

Author: ```suggestion
IMPORTANT: {kib} has a new logging system in 8.0 and the log formats have changed. 
```

> For additional information, see {kibana-ref}/upgrade.html[configuring logging].

Author: ```suggestion
For additional information, see {kibana-ref}/_upgrading_multiple_kibana_instances.html#logging-config-changes[configuring logging].
```
